src/components/Tooltip/Tooltip.tsx

```diff
@@ -56,113 +59,268 @@ const paddingsBottom = {
     large: "tw-pb-4",
 };
 
-const TooltipComponent: ForwardRefRenderFunction<HTMLDivElement, TooltipProps> = (
-    {
-        content,
-        tooltipIcon,
-        heading,
-        headingIcon,
-        linkUrl,
-        linkLabel,
-        brightHeader,
-        buttons,
-        tooltipAriaProps = {},
-        style,
-        children,
-        popperAttributes,
-    },
-    tooltipContainerRef,
-) => {
+export enum TooltipPosition {
+    Top = "top",
+    Right = "right",
+    Bottom = "bottom",
+    Left = "left",
+}
+
+export enum TooltipAlignment {
+    Start = "Start",
+    Middle = "Middle",
+    End = "End",
+}
+
+const placementMap: Record<string, VariationPlacement> = {
+    ["top-Start"]: "top-start",
+    ["top-End"]: "top-end",
+    ["bottom-Start"]: "bottom-start",
+    ["bottom-End"]: "bottom-end",
+    ["left-Start"]: "left-start",
+    ["left-End"]: "left-end",
+    ["right-Start"]: "right-start",
+    ["right-End"]: "right-end",
+};
+
+export const Tooltip: FC<TooltipProps> = ({
```

Comment:
[`FC` is not allowed anymore](https://frontify.slack.com/archives/C01JE9QENCF/p1650544457958619)

Reply:
True! And I voted to keep it 😁 . I removed it. Thanks for all the feedback. I took into account all your comments and I just simplified everything again.